templates/.github/workflows/js-test-and-release.yml

```
@@ -137,6 +137,9 @@ jobs:
         with:
           docker-token: ${{ secrets.DOCKER_TOKEN }}
           docker-username: ${{ secrets.DOCKER_USERNAME }}
+      - run: npm run --if-present docs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: npm run --if-present release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Run docs after release otherwise the version number on the generated docs is always one version out of date.

```suggestion
      - run: npm run --if-present release
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      - run: npm run --if-present docs
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```